cmd/testrunner.go

```diff
@@ -534,6 +535,15 @@ func testRunnerSystemCommandAction(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("can't create Kibana client: %w", err)
 	}
+	versionInfo, err := kibanaClient.Version()
+	if err != nil {
+		return fmt.Errorf("can't get version info from Kibana client: %w", err)
+	}
+	stackVersion, err := semver.NewVersion(versionInfo.Number)
+	if err != nil {
+		return fmt.Errorf("can't parse Kibana version %q: %w", versionInfo.Number, err)
+	}
+	checkFailureStore := !stackVersion.LessThan(semver.MustParse("8.14.0"))
```

Comment:
We should not rely only on the version, but actually check that ES supports the failure store.